tekton/release-pipeline.yaml

```diff
@@ -28,6 +28,8 @@ spec:
     params:
       - name: package
         value: $(params.package)
+      - name: packages
+        value: "`go list ./... | grep -v test/e2e`"
```

Comment:
Interesting :stuck_out_tongue: 
On `pipeline` (and `cli`, and others), we do use `e2e` buildtag for the e2e tests so that we don't have to do this :upside_down_face: 

This can be fixed later though :wink: (but worth an issue :stuck_out_tongue:)

Reply:
nice, i have added the `e2e` build tag and reset the pipeline